<launch>

	<group ns="stereo">
		

		<!-- <group ns="left">
			<node name="image_proc" pkg="image_proc" type="image_proc"/>
		</group> -->

		<!-- <group ns="right">
			<node name="image_proc" pkg="image_proc" type="image_proc"/>
		</group>  -->


		<node name="left" pkg="avt_vimba_camera" type="mono_camera_node" output="screen">
			<param name="guid" value="50-0536883838"/>
			<param name="ip_address" value="192.168.1.33"/>
			<param name="camera_info_url" value="file://$(find avt_vimba_camera)/calibrations/calibration_50-0503343289.yaml"/>
			<param name="frame_id" value="left_optical"/>
			<param name="trig_timestamp_topic" value=""/>
			<param name="trigger_source" value="FixedRate"/>
			<!--param name="sync_out_source" value="Acquiring"/-->
			<param name="sync_out_source" value="Exposing"/>
			<param name="sync_out_polarity" value="Normal"/>
			<param name="acquisition_mode" value="Continuous"/>
			<param name="acquisition_rate" value="10"/>
			<param name="pixel_format" value="BayerRG8"/>
			<param name="exposure" value="4000"/>
			<param name="auto_exposure" value="Off"/>
			<param name="gain" value="24"/>
			<param name="auto_gain" value="Off"/>
			<param name="auto_whitebalance" value="Auto"/>
			<param name="x_offset" value="8"/>
			<param name="y_offset" value="9"/>
			<param name="width" value="1920"/>
			<param name="height" value="1440"/>
			<param name="ptp_mode" value="Master"/>
			<param name="stream_bytes_per_second" value="45000000"/>
		</node>

		<node name="right" pkg="avt_vimba_camera" type="mono_camera_node" output="screen">
			<param name="guid" value="50-0536877041"/>
			<param name="ip_address" value="192.168.1.34"/>
			<param name="camera_info_url" value="file://$(find avt_vimba_camera)/calibrations/calibration_50-0503343290.yaml"/>
			<param name="frame_id" value="right_optical"/>
			<param name="trig_timestamp_topic" value=""/>
			<param name="trigger_source" value="Line1"/>
			<param name="trigger_selector" value="AcquisitionStart"/>
			<param name="trigger_mode" value="On"/>
			<param name="sync_in_selector" value="SyncIn1"/>
			<param name="acquisition_mode" value="Continuous"/>
			<param name="pixel_format" value="BayerRG8"/>
			<param name="exposure" value="4000"/>
			<param name="auto_exposure" value="Off"/>
			<param name="gain" value="24"/>
			<param name="auto_gain" value="Off"/>
			<param name="auto_whitebalance" value="Auto"/>
			<param name="x_offset" value="8"/>
			<param name="y_offset" value="9"/>
			<param name="width" value="1920"/>
			<param name="height" value="1440"/>
			<param name="ptp_mode" value="Slave"/>
			<param name="stream_bytes_per_second" value="45000000"/>
		</node>
	</group>
</launch>

